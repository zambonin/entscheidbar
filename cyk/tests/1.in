S
SAB
ab
S -> AB
A -> a
B -> b
# -> #
ab
a
#
S
SAB
ab
S -> AB
A -> a
B -> b
S -> a
# -> #
ab
a
#
S
S
z
# -> #
#
S
S
cab
S -> a
S -> b
# -> #
a
b
c
#
S
SABC
ab
S -> AB
S -> BC
A -> BA
A -> a
B -> CC
B -> b
C -> AB
C -> a
# -> #
baaba
ab
abaa
a
aaaaa
bbbbb
#
S
SABC
ab
S -> AB
S -> BC
A -> BA
A -> a
B -> CC
B -> b
C -> AB
# -> #
baaba
ab
abaa
a
aaaaa
bbbbb
#
S
SAB
ab
S -> AB
A -> AA
A -> a
B -> b
# -> #
ab
aaab
aba
baaaaaaaa
abbbbbb
aaaaaba
baaaaaaaab
aaaa
a
ab
#
S
SAB
ab
S -> AB
B -> b
# -> #
ab
aaab
aba
baaaaaaaa
abbbbbb
aaaaaba
baaaaaaaab
aaaa
a
ab
#
S
POIUYTREWQASDFGHJKLMNBVCXZ
poiuytrewqasdfghjklmnbvcxz
S -> AS
A -> a
S -> BS
B -> b
S -> CS
C -> c
S -> DS
D -> d
S -> ES
E -> e
S -> FS
F -> f
S -> GS
G -> g
S -> HS
H -> h
S -> IS
I -> i
S -> JS
J -> j
S -> KS
K -> k
S -> LS
L -> l
S -> MS
M -> m
S -> NS
N -> n
S -> OS
O -> o
S -> PS
P -> p
S -> QS
Q -> q
S -> RS
S -> SS
R -> r
S -> TS
T -> t
S -> US
U -> u
S -> VS
V -> v
S -> XS
X -> x
S -> WS
W -> w
S -> YS
Y -> y
S -> ZS
Z -> z
S -> a
S -> b
S -> c
S -> d
S -> e
S -> f
S -> g
S -> h
S -> i
S -> j
S -> k
S -> l
S -> m
S -> n
S -> o
S -> p
S -> q
S -> r
S -> s
S -> t
S -> u
S -> v
S -> w
S -> x
S -> y
S -> z
# -> #
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
poiuytrewqasdfghjklmnbvcxz
zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz
abc
cba
hlashfluahfreubyvwaruiewyfuaiwbyfwaelbfyewbfyefawk
alnceuyafcbuearyvuiawefyeulawybrebydaywauyeryewyew
laeucybeakufdyesuidyndfyyelquiweyfreylaseuyfieslfy
awnerxyaewyfqpriuoewiahlsfjkhadsfjkhdasflhfldshajj
q
w
e
r
t
y
u
i
o
p
a
s
d
f
g
h
j
k
l
z
x
c
v
b
n
m
#
S
POIUYTREWQASDFGHJKLMNBVCXZ
poiuytrewqasdfghjklmnbvcxz
S -> AS
A -> a
S -> BS
B -> b
S -> CS
C -> c
S -> DS
D -> d
S -> ES
E -> e
S -> FS
F -> f
S -> GS
G -> g
S -> HS
H -> h
S -> IS
I -> i
S -> JS
J -> j
S -> KS
K -> k
S -> LS
L -> l
S -> MS
M -> m
S -> NS
N -> n
S -> OS
O -> o
S -> PS
P -> p
S -> QS
Q -> q
S -> RS
R -> r
S -> TS
T -> t
S -> US
U -> u
S -> VS
V -> v
S -> XS
X -> x
S -> WS
W -> w
S -> YS
Y -> y
S -> ZS
Z -> z
S -> a
S -> b
S -> c
S -> d
S -> e
S -> f
S -> g
S -> h
S -> i
S -> j
S -> k
S -> l
S -> m
S -> n
S -> o
S -> p
S -> q
S -> r
S -> s
S -> t
S -> u
S -> v
S -> w
S -> x
S -> y
S -> z
# -> #
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
poiuytrewqasdfghjklmnbvcxz
zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz
abc
cba
hlashfluahfreubyvwaruiewyfuaiwbyfwaelbfyewbfyefawk
alnceuyafcbuearyvuiawefyeulawybrebydaywauyeryewyew
laeucybeakufdyesuidyndfyyelquiweyfreylaseuyfieslfy
awnerxyaewyfqpriuoewiahlsfjkhadsfjkhdasflhfldshajj
q
w
e
r
t
y
u
i
o
p
a
s
d
f
g
h
j
k
l
z
x
c
v
b
n
m
#
S
SAB
ab
S -> AB
A -> a
B -> b
# -> #
ab
a
#
S
SAB
ab
S -> AB
A -> a
B -> b
S -> a
# -> #
ab
a
#
S
S
cab
S -> a
S -> b
# -> #
a
b
c
#
S
SABC
ab
S -> AB
S -> BC
A -> BA
A -> a
B -> CC
B -> b
C -> AB
C -> a
# -> #
baaba
ab
abaa
a
aaaaa
bbbbb
#
S
SABC
ab
S -> AB
S -> BC
A -> BA
A -> a
B -> CC
B -> b
C -> AB
# -> #
baaba
ab
abaa
a
aaaaa
bbbbb
#
S
SAB
ab
S -> AB
A -> AA
A -> a
B -> b
# -> #
ab
aaab
aba
baaaaaaaa
abbbbbb
aaaaaba
baaaaaaaab
aaaa
a
ab
#
S
SAB
ab
S -> AB
B -> b
# -> #
ab
aaab
aba
baaaaaaaa
abbbbbb
aaaaaba
baaaaaaaab
aaaa
a
ab
#
S
POIUYTREWQASDFGHJKLMNBVCXZ
poiuytrewqasdfghjklmnbvcxz
S -> AS
A -> a
S -> BS
B -> b
S -> CS
C -> c
S -> DS
D -> d
S -> ES
E -> e
S -> FS
F -> f
S -> GS
G -> g
S -> HS
H -> h
S -> IS
I -> i
S -> JS
J -> j
S -> KS
K -> k
S -> LS
L -> l
S -> MS
M -> m
S -> NS
N -> n
S -> OS
O -> o
S -> PS
P -> p
S -> QS
Q -> q
S -> RS
S -> SS
R -> r
S -> TS
T -> t
S -> US
U -> u
S -> VS
V -> v
S -> XS
X -> x
S -> WS
W -> w
S -> YS
Y -> y
S -> ZS
Z -> z
S -> a
S -> b
S -> c
S -> d
S -> e
S -> f
S -> g
S -> h
S -> i
S -> j
S -> k
S -> l
S -> m
S -> n
S -> o
S -> p
S -> q
S -> r
S -> s
S -> t
S -> u
S -> v
S -> w
S -> x
S -> y
S -> z
# -> #
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
poiuytrewqasdfghjklmnbvcxz
zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz
abc
cba
hlashfluahfreubyvwaruiewyfuaiwbyfwaelbfyewbfyefawk
alnceuyafcbuearyvuiawefyeulawybrebydaywauyeryewyew
laeucybeakufdyesuidyndfyyelquiweyfreylaseuyfieslfy
awnerxyaewyfqpriuoewiahlsfjkhadsfjkhdasflhfldshajj
q
w
e
r
t
y
u
i
o
p
a
s
d
f
g
h
j
k
l
z
x
c
v
b
n
m
#
S
POIUYTREWQASDFGHJKLMNBVCXZ
poiuytrewqasdfghjklmnbvcxz
S -> AS
A -> a
S -> BS
B -> b
S -> CS
C -> c
S -> DS
D -> d
S -> ES
E -> e
S -> FS
F -> f
S -> GS
G -> g
S -> HS
H -> h
S -> IS
I -> i
S -> JS
J -> j
S -> KS
K -> k
S -> LS
L -> l
S -> MS
M -> m
S -> NS
N -> n
S -> OS
O -> o
S -> PS
P -> p
S -> QS
Q -> q
S -> RS
R -> r
S -> TS
T -> t
S -> US
U -> u
S -> VS
V -> v
S -> XS
X -> x
S -> WS
W -> w
S -> YS
Y -> y
S -> ZS
Z -> z
S -> a
S -> b
S -> c
S -> d
S -> e
S -> f
S -> g
S -> h
S -> i
S -> j
S -> k
S -> l
S -> m
S -> n
S -> o
S -> p
S -> q
S -> r
S -> s
S -> t
S -> u
S -> v
S -> w
S -> x
S -> y
S -> z
# -> #
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
poiuytrewqasdfghjklmnbvcxz
zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz
abc
cba
hlashfluahfreubyvwaruiewyfuaiwbyfwaelbfyewbfyefawk
alnceuyafcbuearyvuiawefyeulawybrebydaywauyeryewyew
laeucybeakufdyesuidyndfyyelquiweyfreylaseuyfieslfy
awnerxyaewyfqpriuoewiahlsfjkhadsfjkhdasflhfldshajj
q
w
e
r
t
y
u
i
o
p
a
s
d
f
g
h
j
k
l
z
x
c
v
b
n
m
#
S
S
z
# -> #
z
#
S
S
a
# -> #
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas
poiuytrewqasdfghjklmnbvcxz
zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz
abc
cba
hlashfluahfreubyvwaruiewyfuaiwbyfwaelbfyewbfyefawk
alnceuyafcbuearyvuiawefyeulawybrebydaywauyeryewyew
laeucybeakufdyesuidyndfyyelquiweyfreylaseuyfieslfy
awnerxyaewyfqpriuoewiahlsfjkhadsfjkhdasflhfldshajj
q
w
e
r
t
y
u
i
o
p
a
s
d
f
g
h
j
k
l
z
x
c
v
b
n
m
#
S
S
T.
S -> T
S -> .
# -> #
T
.
.T
#
