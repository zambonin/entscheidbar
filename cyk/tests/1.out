Instancia 1
ab e uma palavra valida
a nao e uma palavra valida

Instancia 2
ab e uma palavra valida
a e uma palavra valida

Instancia 3

Instancia 4
a e uma palavra valida
b e uma palavra valida
c nao e uma palavra valida

Instancia 5
baaba e uma palavra valida
ab e uma palavra valida
abaa e uma palavra valida
a nao e uma palavra valida
aaaaa e uma palavra valida
bbbbb nao e uma palavra valida

Instancia 6
baaba nao e uma palavra valida
ab e uma palavra valida
abaa nao e uma palavra valida
a nao e uma palavra valida
aaaaa nao e uma palavra valida
bbbbb nao e uma palavra valida

Instancia 7
ab e uma palavra valida
aaab e uma palavra valida
aba nao e uma palavra valida
baaaaaaaa nao e uma palavra valida
abbbbbb nao e uma palavra valida
aaaaaba nao e uma palavra valida
baaaaaaaab nao e uma palavra valida
aaaa nao e uma palavra valida
a nao e uma palavra valida
ab e uma palavra valida

Instancia 8
ab nao e uma palavra valida
aaab nao e uma palavra valida
aba nao e uma palavra valida
baaaaaaaa nao e uma palavra valida
abbbbbb nao e uma palavra valida
aaaaaba nao e uma palavra valida
baaaaaaaab nao e uma palavra valida
aaaa nao e uma palavra valida
a nao e uma palavra valida
ab nao e uma palavra valida

Instancia 9
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas e uma palavra valida
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas e uma palavra valida
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas e uma palavra valida
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas e uma palavra valida
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas e uma palavra valida
poiuytrewqasdfghjklmnbvcxz e uma palavra valida
zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz e uma palavra valida
abc e uma palavra valida
cba e uma palavra valida
hlashfluahfreubyvwaruiewyfuaiwbyfwaelbfyewbfyefawk e uma palavra valida
alnceuyafcbuearyvuiawefyeulawybrebydaywauyeryewyew e uma palavra valida
laeucybeakufdyesuidyndfyyelquiweyfreylaseuyfieslfy e uma palavra valida
awnerxyaewyfqpriuoewiahlsfjkhadsfjkhdasflhfldshajj e uma palavra valida
q e uma palavra valida
w e uma palavra valida
e e uma palavra valida
r e uma palavra valida
t e uma palavra valida
y e uma palavra valida
u e uma palavra valida
i e uma palavra valida
o e uma palavra valida
p e uma palavra valida
a e uma palavra valida
s e uma palavra valida
d e uma palavra valida
f e uma palavra valida
g e uma palavra valida
h e uma palavra valida
j e uma palavra valida
k e uma palavra valida
l e uma palavra valida
z e uma palavra valida
x e uma palavra valida
c e uma palavra valida
v e uma palavra valida
b e uma palavra valida
n e uma palavra valida
m e uma palavra valida

Instancia 10
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas e uma palavra valida
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas e uma palavra valida
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas e uma palavra valida
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas e uma palavra valida
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas e uma palavra valida
poiuytrewqasdfghjklmnbvcxz nao e uma palavra valida
zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz e uma palavra valida
abc e uma palavra valida
cba e uma palavra valida
hlashfluahfreubyvwaruiewyfuaiwbyfwaelbfyewbfyefawk nao e uma palavra valida
alnceuyafcbuearyvuiawefyeulawybrebydaywauyeryewyew e uma palavra valida
laeucybeakufdyesuidyndfyyelquiweyfreylaseuyfieslfy nao e uma palavra valida
awnerxyaewyfqpriuoewiahlsfjkhadsfjkhdasflhfldshajj nao e uma palavra valida
q e uma palavra valida
w e uma palavra valida
e e uma palavra valida
r e uma palavra valida
t e uma palavra valida
y e uma palavra valida
u e uma palavra valida
i e uma palavra valida
o e uma palavra valida
p e uma palavra valida
a e uma palavra valida
s e uma palavra valida
d e uma palavra valida
f e uma palavra valida
g e uma palavra valida
h e uma palavra valida
j e uma palavra valida
k e uma palavra valida
l e uma palavra valida
z e uma palavra valida
x e uma palavra valida
c e uma palavra valida
v e uma palavra valida
b e uma palavra valida
n e uma palavra valida
m e uma palavra valida

Instancia 11
ab e uma palavra valida
a nao e uma palavra valida

Instancia 12
ab e uma palavra valida
a e uma palavra valida

Instancia 13
a e uma palavra valida
b e uma palavra valida
c nao e uma palavra valida

Instancia 14
baaba e uma palavra valida
ab e uma palavra valida
abaa e uma palavra valida
a nao e uma palavra valida
aaaaa e uma palavra valida
bbbbb nao e uma palavra valida

Instancia 15
baaba nao e uma palavra valida
ab e uma palavra valida
abaa nao e uma palavra valida
a nao e uma palavra valida
aaaaa nao e uma palavra valida
bbbbb nao e uma palavra valida

Instancia 16
ab e uma palavra valida
aaab e uma palavra valida
aba nao e uma palavra valida
baaaaaaaa nao e uma palavra valida
abbbbbb nao e uma palavra valida
aaaaaba nao e uma palavra valida
baaaaaaaab nao e uma palavra valida
aaaa nao e uma palavra valida
a nao e uma palavra valida
ab e uma palavra valida

Instancia 17
ab nao e uma palavra valida
aaab nao e uma palavra valida
aba nao e uma palavra valida
baaaaaaaa nao e uma palavra valida
abbbbbb nao e uma palavra valida
aaaaaba nao e uma palavra valida
baaaaaaaab nao e uma palavra valida
aaaa nao e uma palavra valida
a nao e uma palavra valida
ab nao e uma palavra valida

Instancia 18
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas e uma palavra valida
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas e uma palavra valida
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas e uma palavra valida
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas e uma palavra valida
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas e uma palavra valida
poiuytrewqasdfghjklmnbvcxz e uma palavra valida
zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz e uma palavra valida
abc e uma palavra valida
cba e uma palavra valida
hlashfluahfreubyvwaruiewyfuaiwbyfwaelbfyewbfyefawk e uma palavra valida
alnceuyafcbuearyvuiawefyeulawybrebydaywauyeryewyew e uma palavra valida
laeucybeakufdyesuidyndfyyelquiweyfreylaseuyfieslfy e uma palavra valida
awnerxyaewyfqpriuoewiahlsfjkhadsfjkhdasflhfldshajj e uma palavra valida
q e uma palavra valida
w e uma palavra valida
e e uma palavra valida
r e uma palavra valida
t e uma palavra valida
y e uma palavra valida
u e uma palavra valida
i e uma palavra valida
o e uma palavra valida
p e uma palavra valida
a e uma palavra valida
s e uma palavra valida
d e uma palavra valida
f e uma palavra valida
g e uma palavra valida
h e uma palavra valida
j e uma palavra valida
k e uma palavra valida
l e uma palavra valida
z e uma palavra valida
x e uma palavra valida
c e uma palavra valida
v e uma palavra valida
b e uma palavra valida
n e uma palavra valida
m e uma palavra valida

Instancia 19
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas e uma palavra valida
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas e uma palavra valida
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas e uma palavra valida
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas e uma palavra valida
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas e uma palavra valida
poiuytrewqasdfghjklmnbvcxz nao e uma palavra valida
zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz e uma palavra valida
abc e uma palavra valida
cba e uma palavra valida
hlashfluahfreubyvwaruiewyfuaiwbyfwaelbfyewbfyefawk nao e uma palavra valida
alnceuyafcbuearyvuiawefyeulawybrebydaywauyeryewyew e uma palavra valida
laeucybeakufdyesuidyndfyyelquiweyfreylaseuyfieslfy nao e uma palavra valida
awnerxyaewyfqpriuoewiahlsfjkhadsfjkhdasflhfldshajj nao e uma palavra valida
q e uma palavra valida
w e uma palavra valida
e e uma palavra valida
r e uma palavra valida
t e uma palavra valida
y e uma palavra valida
u e uma palavra valida
i e uma palavra valida
o e uma palavra valida
p e uma palavra valida
a e uma palavra valida
s e uma palavra valida
d e uma palavra valida
f e uma palavra valida
g e uma palavra valida
h e uma palavra valida
j e uma palavra valida
k e uma palavra valida
l e uma palavra valida
z e uma palavra valida
x e uma palavra valida
c e uma palavra valida
v e uma palavra valida
b e uma palavra valida
n e uma palavra valida
m e uma palavra valida

Instancia 20
z nao e uma palavra valida

Instancia 21
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas nao e uma palavra valida
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas nao e uma palavra valida
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas nao e uma palavra valida
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas nao e uma palavra valida
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaas nao e uma palavra valida
poiuytrewqasdfghjklmnbvcxz nao e uma palavra valida
zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz nao e uma palavra valida
abc nao e uma palavra valida
cba nao e uma palavra valida
hlashfluahfreubyvwaruiewyfuaiwbyfwaelbfyewbfyefawk nao e uma palavra valida
alnceuyafcbuearyvuiawefyeulawybrebydaywauyeryewyew nao e uma palavra valida
laeucybeakufdyesuidyndfyyelquiweyfreylaseuyfieslfy nao e uma palavra valida
awnerxyaewyfqpriuoewiahlsfjkhadsfjkhdasflhfldshajj nao e uma palavra valida
q nao e uma palavra valida
w nao e uma palavra valida
e nao e uma palavra valida
r nao e uma palavra valida
t nao e uma palavra valida
y nao e uma palavra valida
u nao e uma palavra valida
i nao e uma palavra valida
o nao e uma palavra valida
p nao e uma palavra valida
a nao e uma palavra valida
s nao e uma palavra valida
d nao e uma palavra valida
f nao e uma palavra valida
g nao e uma palavra valida
h nao e uma palavra valida
j nao e uma palavra valida
k nao e uma palavra valida
l nao e uma palavra valida
z nao e uma palavra valida
x nao e uma palavra valida
c nao e uma palavra valida
v nao e uma palavra valida
b nao e uma palavra valida
n nao e uma palavra valida
m nao e uma palavra valida

Instancia 22
T e uma palavra valida
. e uma palavra valida
.T nao e uma palavra valida

